### PR TITLE
 DBMS - Tutorial or courses

### LINKED ISSUE #294: Add: DBMS Resources
I wish to add some links to free courses

### PR DESCRIPTION
**Pull request by sau-mili**
**IMPORTANT:** When creating your Pull Request, please use the following format for the title: `Category - Resource Name & Section`  

PR Title samples:

- `Web development - CSS Portal & Tools`
- `Web development - Leetcode & Challenges`

### Fixes Issue

fixes #294 

### Description: 

Added two resources of DBMS tutorials or courses

### Resource URL: 

https://www.youtube.com/watch?v=ExcRbA7fy_A&list=PL4cUxeGkcC9h77dJ-QJlwGlZlTd4ecZOA&ab_channel=NetNinja
https://www.udemy.com/course/database-engines-crash-course/?couponCode=LEADERSALE24A

### Checklist:
Before submitting your pull request, ensure that you have completed the following tasks: 

- [x] I have carefully reviewed and adhered to the [contributing guidelines](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md) before creating this pull request.
- [x] I followed the prescribed PR title template. _(Check this if you're adding a resource)_

### CODE REVIEW

File: DBMS (Database)/README.md [hunk 1]
```diff
@@ -103,6 +102,14 @@ A database is an organized collection of structured information, or data, typica
       <td><a href="https://www.edx.org/learn/databases">edX</a></td>
       <td>Offers Courses on database development and database queries.</td>
     </tr>
+  <tr>
+      <td><a href="https://shorturl.at/6oNur">NetNinja Youtube</a></td>
```
Comment (jfmartinz) [inline DBMS (Database)/README.md:106]: Thhe URL is 404, not found.

**Review by jfmartinz — CHANGES_REQUESTED** [review]

**Comment by sau-mili** [comment]
Can you check now?

**Review by jfmartinz — APPROVED** [review]
Looks good @sau-mili

**Comment by jfmartinz** [comment]
Wait you need to resolve the conflict, this link might help: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github

**Comment by sau-mili** [comment]
But, in this pull req, it says I can't resolve it because I Dont have writing access
Screenshot 2024-05-19 122621

**Comment by sau-mili** [comment]
@jfmartinz please check this. I have raised more issues

**Comment by jfmartinz** [comment]
Resolve Conflict first, thnaks

**Comment by sau-mili** [comment]
Please check the screenshot, resolve button is greyed out for me

**Comment by sau-mili** [comment]
Should I close the pull and redo it altogether?

**Comment by jfmartinz** [comment]
I cant merged it because there is conflict on the changes you made. I sent you a link, what it is already, you can read it. Its grey means that you need to resolve first before that PR be able to merged into main. 

yes you can redo it instead. Just dont forget to pull before create changes so that eveything is up to date and prevent conflict, thanks

**Comment by sau-mili** [comment]
#343 redo done

**Pull request closed without merging**